sequana/kraken.py

```diff
@@ -481,7 +484,10 @@ def plot2(self, kind="pie", fontsize=12):
             zlabels += list(z1.ratio.index) + [''] * len(z2.ratio)
             Z.extend(z)
 
-            labels.append(kingdom)
+            if kingdom.strip() == "":
```

Comment:
Tu peux juste faire `if kingdom.strip():`

Reply:
okay for next PR